vault/resource_identity_oidc_provider.go

```diff
@@ -0,0 +1,211 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+	"net/url"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+const identityOIDCProviderPathPrefix = "identity/oidc/provider"
+
+func identityOIDCProviderResource() *schema.Resource {
+	return &schema.Resource{
+		Create: identityOIDCProviderCreateUpdate,
+		Update: identityOIDCProviderCreateUpdate,
+		Read:   identityOIDCProviderRead,
+		Delete: identityOIDCProviderDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				ForceNew:    true,
+				Description: "The name of the provider.",
+				Required:    true,
+			},
+			"issuer": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Description: "Specifies what will be used as the 'scheme://host:port' component for the 'iss' claim of ID tokens." +
+					"This value is computed using the issuer_host and https_enabled schema fields.",
+				Computed: true,
+				// TODO confirm if this is needed
+				Optional: true,
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Probably don't need this since you changed the way that issuer is configured.

Reply:
Removed in 68a0eb6